### PR TITLE
chore: Next 14 to Next 14.1.0

### PR DESCRIPTION
**Pull request by tihuan**
## Reason for Change

- #TICKET_NUMBER
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

- add
- remove
- modify

## Testing steps

- Either list QA steps or reasoning you feel QA is unnecessary
- Describe how you made sure to know that your changes worked. Should allow someone else to go verify your code without in depth knowledge.
- "Unit tested only", "tested in rdev by a, b, c, verifying feature worked by... ", "manually ran pipeline locally with these results: ..."

## Checklist 🛎️

- [ ] Add product, design, and eng as reviewers for rdev review
- [ ] For UI changes, add screenshots/videos, so the reviewers know what you expect them to see
- [ ] For UI changes, add e2e tests to prevent regressions

### CODE REVIEW

**Comment by tihuan** [comment]
A quick PR to upgrade to Next 14.1.0, which came out yesterday at 3pm lol

**Review by kaloster — APPROVED** [review]

File: frontend/tests/features/wheresMyGene/wheresMyGene.test.ts [hunk 1]
```diff
@@ -408,22 +408,20 @@ describe("Where's My Gene", () => {
...
+    // Note: This test could fail if we find more marker genes for the targeted cell.
     // If this happens, just mark the test as skipped and ping @joyceyan
     test(`Should verify copy for cell types with no marker genes`, async ({
       page,
     }) => {
       await goToPage(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`, page);
 
       // Expand embryo tissue
-      await expandTissue(page, "embryo");
+      await expandTissue(page, "saliva");
```
Comment (tihuan) [inline frontend/tests/features/wheresMyGene/wheresMyGene.test.ts:419]: @joyceyan the updated tissue and cell for the test are here!

Comment (joyceyan) [reply frontend/tests/features/wheresMyGene/wheresMyGene.test.ts:419]: weird - this change seems fine though!

Comment (tihuan) [reply frontend/tests/features/wheresMyGene/wheresMyGene.test.ts:419]: Awesome! Yeah I checked that saliva leukocyte doesn't have marker genes in staging and prod too 👍

**Pull request merged**